pyvrp/tests/test_Individual.py

```diff
@@ -269,16 +261,17 @@ def test_str_contains_essential_information():
                 assert_(str(client) in str_route)
 
         # Last line should contain the cost
-        assert_(str(individual.cost()) in str_representation[-1])
+        # TODO what do we want with cost in str representation?
+        # assert_(str(individual.cost(pm)) in str_representation[-1])
```

Comment:
Presumably keep it in there as-is, so leave this test intact. If desired we can open an issue to discuss it, but I think VRPLIB requires the cost to be in the (VRPLIB) format that we currently print the Individual as. But @leonlan knows that better than I do.

Reply:
I agree we should ideally have it in here but that is not simply possible as the `__str__` function cannot rely on the penalty manager, so I'll leave this for now. We can discuss it further in #100.